beams/setup.py

```diff
@@ -799,12 +922,336 @@ def get_job_applicant_custom_fields():
+            },
+            {
+                "fieldname": "interviewed_location",
+                "fieldtype": "Data",
+                "label": "Location",
+                "insert_after": "interviewed_column_break"
+            },
+            {
+                "fieldname": "interviewed_outcome",
+                "fieldtype": "Data",
+                "label": "Outcome",
+                "insert_after": "interviewed_location"
+            },
+            {
+                "fieldname": "travel_session_break",
+                "fieldtype": "Section Break",
+                "label": "Are you willing to travel :",
+                "insert_after": "interviewed_outcome"
+            },
+            {
+                "fieldname": "in_india",
+                "fieldtype": "Check",
+                "label": "In India",
+                "insert_after": "travel_session_break"
+            },
+            {
+                "fieldname": "state_restriction",
+                "fieldtype": "Data",
+                "label": "State Restriction If any",
+                "insert_after": "in_india"
+            },
+            {
+                "fieldname": "india_column_break",
+                "fieldtype": "Column Break",
+                "label": "",
+                "insert_after": "state_restriction"
+            },
+            {
+                "fieldname": "abroad",
+                "fieldtype": "Check",
+                "label": "Abroad",
+                "insert_after": "india_column_break"
+            },
+            {
+                "fieldname": "related_session_break",
+                "fieldtype": "Section Break",
+                "label": "Are you related to any of employee of the Madhyamam Group? If yes,please give details :",
+                "insert_after": "state_restriction"
+            },
+            {
+                "fieldname": "name_of_related_employee",
+                "fieldtype": "Data",
+                "label": "Name",
+                "insert_after": "related_session_break"
+            },
+            {
+                "fieldname": "name_of_related_employee_org",
+                "fieldtype": "Data",
+                "label": "Organization",
+                "insert_after": "name_of_related_employee"
+            },
+            {
+                "fieldname": "related_column_break",
+                "fieldtype": "Column Break",
+                "label": "",
+                "insert_after": "name_of_related_employee_org"
+            },
+            {
+                "fieldname": "name_of_related_employee_pos",
+                "fieldtype": "Data",
+                "label": "Position",
+                "insert_after": "related_column_break",
+            },
+            {
+                "fieldname": "name_of_related_employee_rel",
+                "fieldtype": "Data",
+                "label": "Relationship",
+                "insert_after": "name_of_related_employee_pos"
```

Comment:
fieldnames doesn't make much sense, name_of suffix is unnecessary. Keep in mind for future